Gems/GradientSignal/Code/Source/Components/ImageGradientComponent.cpp

```diff
@@ -550,6 +579,56 @@ namespace GradientSignal
         SetupMultiplierAndOffset(m_configuration.m_scaleRangeMin, m_configuration.m_scaleRangeMax);
     }
 
+    float ImageGradientComponent::GetValueForSamplingType(AZ::u32 x, AZ::u32 y, float pixelX, float pixelY) const
+    {
+        float value = 0.0f;
+
+        switch (m_currentSamplingType)
+        {
+        case SamplingType::Point:
+            // Retrieve the pixel value for the single point
+            value = GetPixelValue(x, y);
```

Comment:
Nit: It might be better to just return here and also give a default branch to use point sampling (since point is by default) instead of given a value 0.0.

Reply:
Good call, updated!